lib/dradis/plugins/burp/html/importer.rb

```diff
@@ -90,7 +90,7 @@ def process_html_evidence(html_evidence, issue)
         evidence_id = html_evidence.attr('id').value
         logger.info { "Processing evidence #{evidence_id}" }
 
-        host_td    = html_evidence.xpath("//td[starts-with(.,'Host:')]").first
+        host_td    = html_evidence.xpath(".//td[starts-with(.,'Host:')]").first
```

Comment:
Since we're only looking for the first node why don't we use `at_xpath` instead? 

```suggestion
        host_td    = html_evidence.at_xpath(".//td[starts-with(.,'Host:')]")
```

This may also improve performance in larger uploads